src/components/Icon/iconFont.types.ts

```diff
@@ -0,0 +1,6 @@
+export interface IIconProps {
+  icon: 'at' | 'emote' | 'hashtag' | 'link' | 'pin' | 'Gpt' | 'Canva';
+  size?: 'small' | 'large';
+  color?: 'active' | 'unActive';
```

Comment:
Colocar as outras cores tambem

```suggestion
  color?: 'active' | 'disabled';
```

Reply:
Feito!